creator/ingest_runs/tasks/validation_run.py

```diff
@@ -94,6 +91,17 @@ def validate_file_versions(validation_run):
             "to a file type template. No templates were found for any of "
             "the input files."
         )
+
+    # TEMPORARY FOR TESTING
+    import time
+
+    t_end = time.time() + 20
+    progress = ""
+    while time.time() < t_end:
+        time.sleep(1)
+        progress += "."
+        logger.info(progress)
```

Comment:
still want this?

Reply:
Once we're finished testing with frontend I'll remove this